configs/serviceops/types.yml

```diff
@@ -0,0 +1,4 @@
+types:
+- name: global/types/GenericListCheckerType
```

Comment:
Also, given that the chunks of config are defined within one of these namespaces, I don't think we need to utter the namespace in the name fields, they're implied. And we can make it so not uttering the namespace in a reference (like the template field below, then we look in the current namespace and up to the global namespace automatically.

This means that in general, you don't utter namespaces in your config.